common-instancetypes/instancetypes/hyperscale/cx/1/base/cx1.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: kubevirt.io/v1alpha2
+kind: VirtualMachineInstancetype
```

Comment:
Can we settle for one kind in the base yaml files?

`cx/1/base/cx1.yaml` is VirtualMachineInstancetype
`cx/base/cx.yaml` is VirtualMachineClusterInstancetype